code/game/machinery/scp_294.dm

```diff
@@ -18,8 +18,7 @@
 	dispensable_reagents = null
 	var/list/prohibited_reagents = list(ADMINORDRAZINE)
 
-	machine_flags = FIXED2WORK
-	mech_flags = MECH_SCAN_FAIL
```

Comment:
Why'd you get rid of `MECH_SCAN_FAIL`?  Trying to sneak one past?

Reply:
I'm not entirely sure whether this is sarcasm but this is what it was before.

Reply:
it was part of the reverted PR, made scans fail